src/modules/users/user.entity.ts

```diff
@@ -262,6 +262,13 @@ export class User {
   @Column({ type: 'int', default: 0 })
   orders: number;
 
+  @ApiProperty({
+    example: false,
+    description: 'Indicates if the user is followed',
+  })
+  @Column({ default: false })
+  isFollowed?: boolean;
+
```

Comment:
We need to create a table (many-to-many relationship) where each buyer can follow multiple vendors. This should not be just a field isFollowed in the vendors table.